### PR TITLE
=htt Remove redundant modifiers

### PR DESCRIPTION
**Pull request by naferx**

### CODE REVIEW

File: akka-http-core/src/main/java/akka/http/javadsl/model/headers/LanguageRange.java [hunk 1]
```diff
@@ -5,11 +5,11 @@
...
-    public abstract String primaryTag();
-    public abstract float qValue();
-    public abstract boolean matches(Language language);
-    public abstract Iterable<String> getSubTags();
-    public abstract LanguageRange withQValue(float qValue);
+    String primaryTag();
+    float qValue();
+    boolean matches(Language language);
+    Iterable<String> getSubTags();
+    LanguageRange withQValue(float qValue);
```
Comment (rkuhn) [inline akka-http-core/src/main/java/akka/http/javadsl/model/headers/LanguageRange.java]: aren’t these package-private now?

Comment (ktoso) [inline akka-http-core/src/main/java/akka/http/javadsl/model/headers/LanguageRange.java]: they're public (welcome to java (method declarations in in interface)).

I would like to keep the modifiers, as we have many scala contributors, and after some years of Scala one simply forgets what "default" means in some places in java. 
So thanks for the PR but I'd like to keep them (i.e. not accept the PR), thanks!

Comment (rkuhn) [inline akka-http-core/src/main/java/akka/http/javadsl/model/headers/LanguageRange.java]: right, you may take my confusion as a data point :-)

**Comment by ktoso** [comment]
Thanks for the idea / PR but we do want to be explicit about these things as commented above.
Thanks again and please don't let this discourage you from future contribs, thanks a lot in advance!

**Pull request closed without merging**